### ISSUE TITLE
Notice "Undefined offset: 42" in protected/extensions/curl/Curl.php(32) when CURLOPT_HEADER option not set in extension config

### ISSUE THREAD

**Issue by dbykadorov** [hackerone/curl#12]
Screen http://joxi.ru/GxW0UxjKTJCvNZQ0RYo

Options:

``` php
        // Curl wrapper extension
        'curl' => array(
            'class' => 'ext.curl.Curl',
            'options' => array(
                /**/
            ),
        ),
```

**Comment by hackerone** [comment]
fixed in v 1.1

**Comment by dbykadorov** [comment]
@hackerone thanks!
